gokart/file_processor.py

```diff
@@ -130,7 +130,7 @@ def load(self, file):
     def dump(self, obj, file):
         assert isinstance(obj, (pd.DataFrame, pd.Series)), \
             f'requires pd.DataFrame or pd.Series, but {type(obj)} is passed.'
-        obj.to_csv(file, index=False, sep=self._sep, header=True)
+        obj.to_csv(file, mode='wt', index=False, sep=self._sep, header=True)
```

Comment:
ref: 

https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.2.0.html#support-for-binary-file-handles-in-to-csv

Reply:
https://github.com/spotify/luigi/blob/4d0576c7c265afcb228097af79f316ba0de0242c/luigi/target.py#L324
OK, we can't control the mode of `file` from outside of luigi.
Looks good handling :)